### PR TITLE
provide operator upgrade junit results

### PR DESCRIPTION
**Pull request by deads2k**
Provides accurate accounting of individual operators for upgrade and provides durations for how long they take to upgrade.

### CODE REVIEW

**Comment by deads2k** [comment]
/hold

hold until we see the junit results and like them.

File: test/extended/operators/clusteroperator_upgrade.go [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+			)
+		}
+	}
+
+	return ret
+}
+
+func minorVersionFor(version string) string {
+	// someday, if you're here because this product is around this long and for loop ran out of matches, send an email
+	// to deads2k from github because I'm horrified that this code lasted that long
```
Comment (bparees) [inline test/extended/operators/clusteroperator_upgrade.go:37]: seriously. regex. :)

Comment (smarterclayton) [reply test/extended/operators/clusteroperator_upgrade.go:37]: Wow

**Comment by deads2k** [comment]
that's weird.  Target version is empty....  It must get filled in after an upgrade request and then read back out...

**Pull request closed without merging**